src/Workspaces/CSharp/Portable/Indentation/CSharpIndentationService.Indenter.cs

```diff
@@ -96,7 +96,7 @@ private static IndentationResult GetIndentationBasedOnToken(Indenter indenter, S
             {
                 var interpolatedExpression = token.GetAncestor<InterpolatedStringExpressionSyntax>();
                 Contract.ThrowIfNull(interpolatedExpression);
-                if (interpolatedExpression.StringEndToken.IsKind(SyntaxKind.InterpolatedMultiLineRawStringEndToken))
+                if (interpolatedExpression.StringStartToken.IsKind(SyntaxKind.InterpolatedMultiLineRawStringStartToken))
```

Comment:
this is the one line that ever even examined this.  and it was trivial to still work properly without this particular piece of data.